### ISSUE TITLE
[AOSP 9.0 1.21] Bugs Chuwi hi9 air

### ISSUE THREAD

**Issue by AxelJh** [phhusson/treble_experimentations#887]
-  Some apps won't install via package helper but only via playstore
-  Sometime media.process crash
-  Security patch back to august version after reboot
-  bluetooth don't work
-  and in settings won't save gpu option overlay HW disable after reboot system re-enable overlay

will u fix?, I appreciate your job but......

**Comment by proganime1200** [comment]
didyou try to format is it working on older version

**Comment by AxelJh** [comment]
Yes I tried, no result. If someone can check the latest version 9 and fix issues will be appreciated
